### ISSUE TITLE
Form Icon Miss placed in- Contextual Tabs

### ISSUE THREAD

**Issue by MTABSoftwaredev** [Wagnerp/Krypton-NET-5.470#64]
In Ribbon Form Form Icon are after Contextual tabs in ribbon,

![contexual tab](https://user-images.githubusercontent.com/17450367/43354800-2ff21c2a-9270-11e8-8bb9-170a84ac689c.png)

**Comment by Wagnerp** [comment]
I think that I have fixed it in build 633

**Comment by maniX101** [comment]
No, the bug is still there.

![image](https://user-images.githubusercontent.com/5473498/44202047-87b11400-a168-11e8-976f-25778ea167f1.png)

**Comment by Smurf-IV** [comment]
Note:
The Icon only appears in the Demo App once a Resize is triggered, 
It's still in the wrong place 
- [Old Excel](https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=2ahUKEwjc6sycr_fgAhUJ4YUKHb_iAPIQjRx6BAgBEAU&url=https%3A%2F%2Fwww.exceltip.com%2Fbasic-excel%2Fcontextual-tab.html&psig=AOvVaw0Y5-qXwoDiqL40a3_WHzxs&ust=1552300204759495)
- [Word 2010](https://www.google.com/imgres?imgurl=https%3A%2F%2Fi.ytimg.com%2Fvi%2FTPRrACYWrcQ%2Fhqdefault.jpg&imgrefurl=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DTPRrACYWrcQ&docid=VoOtjmyi1IhCJM&tbnid=QbbOM-9eQXcgrM%3A&vet=10ahUKEwjClLGDr_fgAhVConEKHTYJAtoQMwhLKBAwEA..i&w=480&h=360&safe=off&client=firefox-b-d&bih=944&biw=1920&q=contextual%20tabs&ved=0ahUKEwjClLGDr_fgAhVConEKHTYJAtoQMwhLKBAwEA&iact=mrc&uact=8)

Basically, the icon should to the left of the QAT drop down, regardless Because it does not appear to wrap the default from Icon when in Office 2007 or Sparkle mode either!) 
![image](https://user-images.githubusercontent.com/2418812/54083848-ea464900-4320-11e9-898c-11fee5315a34.png)

**Comment by Smurf-IV** [comment]
Why was this closed ?

**Comment by Wagnerp** [comment]
Sorry, thought it was something else, though I thought that Windows Ribbon apps had their icons hidden in the titlebar or put into the 'orb' (2007 style)?
